inspirehep/modules/theme/templates/inspirehep_theme/format/record/Inspire_HTML_detailed_macros.tpl

```diff
@@ -265,27 +265,59 @@
 {% endmacro %}
 
 {% macro record_plots(record) %}
+  {% set vars = {'plotExists': False} %}
```

Comment:
Same as in the other PR, if using the extension `do` simplifies the code I can add it before merging this PR.